index.css

```diff
@@ -0,0 +1,67 @@
+@import url("https://argon-css.netlify.app/components/index.css");
+
+/* header search bar */
+
+.search__bar__container {
+  width: 30rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
Create a utility class for this. As it can be reused.

Reply:
okay will do that